docs/source/resources/define-metadata.rst

```diff
@@ -1928,10 +1923,10 @@ variables are booleans.
    * - osx
      - True if the platform is macOS.
    * - arm64
-     - True if the platform is macOS and the Python architecture
-       is arm64.
+     - True if the platform is either macOS or Windows and the
+       Python architecture is arm64.
    * - unix
-     - True if the platform is either macOS or Linux.
+     - True if the platform is either macOS or Linux or emscripten.
```

Comment:
Not released yet (https://github.com/conda/conda-build/commit/d480c32597d7cb986ec89dbc8152d474b627e404)